public/app/features/dashboard-scene/saving/provisioned/DashboardPreviewBanner.tsx

```diff
@@ -85,7 +86,7 @@ function DashboardPreviewBannerContent({ queryParams, slug, path }: DashboardPre
             <Icon name="external-link-alt" />
           </Stack>
         }
-        onRemove={() => window.open(githubURL, '_blank')}
+        onRemove={() => window.open(sanitizeUrl(githubURL), '_blank')}
```

Comment:
this value comes from our API (the /urls section of the response) but similarly seems safe to sanitize